Workbooks/WorkbooksMetadata.json

```diff
@@ -892,5 +892,20 @@
     "templateRelativePath": "WebApplicationFirewallWAFTypeEvents.json",
     "subtitle": "",
     "provider": "Microsoft"
+  },
+  {
+    "workbookKey": "CyberArkWorkbook",
+    "logoFileName": "CyberArk_Logo.svg",
+    "description": "The CyberArk Syslog connector allows you to easily connect all your CyberArk security solution logs with your Azure Sentinel, to view dashboards, create custom alerts, and improve investigation. Integration between CyberArk and Azure Sentinel makes use of the CEF Data Connector to properly parse and display CyberArk Syslog messages.",
+    "dataTypesDependencies": [ "CommonSecurityLog" ],
+    "dataConnectorsDependencies": [ "CyberArk" ],
+    "previewImagesFileNames": [ "CyberArk_White1.png", "CyberArk_Black1.png" ],
+    "version": "1.1",
+    "title": "CyberArk EPV Events",
+    "templateRelativePath": "CyberArkEPV.json",
+    "subtitle": "",
+    "provider": "CyberArk",
+    "fromTemplateId": "CyberArkWorkbook",
```

Comment:
Hi,

the fields fromTemplateId and $Schema are not supported.

Please remove them